### ISSUE TITLE
line breaks at top appears to be broken

### ISSUE THREAD

**Issue by RealRaven2000** [RealRaven2000/SmartTemplates#212]
The old setting for adding line breaks at the top of the document seems to be broken:

![image](https://user-images.githubusercontent.com/6314674/187716049-dd73d00e-d4d3-4971-aa50-c88e5f56c680.png)

Expected behavior - this would add line breaks to create space on top of the template. Workaround: add line breaks (or better a container with padding) to the top of the template.

**Comment by BobinPlano** [comment]
Will it replace my need to enter <br> & %cursor% in the Template field that I must use now?

I'm looking forward to having this broken feature fixed.

**Comment by RealRaven2000** [comment]
> Will it replace my need to enter & %cursor% in the Template field that I must use now?
> 
> I'm looking forward to having this broken feature fixed.

Yes it will. The breaks in the template can be used as a workaround. The # of breaks at top was a very old school way of solving layout issues in plain text mode and is really just a legacy settings. I will upload a version here once I have a fix available.

A better way of doing  layout  is using paragraphs - they provide margins at top and bottom and make for a much cleaner layout - you only have to remember to only hit Enter once instead of twice. Example for a better template. This example works both for write and reply:

```
<p></p>
<p>Dear %recipient(name)%</p>
<p>%cursor%</p>
<p>Yours sincerely<br>
&nbsp;&nbsp;%identity(name)%</p>
```

Once I upload version, github will notify you and you can test it and leave feedback.

**Comment by RealRaven2000** [comment]
I looked at this behavior in detail, and it has always been like this - the line breaks were added between the template text and the quoted email. I don't really want to change this behavior as it may disrupt the expectation of a lot of existing users. 

I would rather correct the label of the original version (which was originally written over 10 years ago by it's then Japanese author) and rename it "Insert distance between reply and quoted part", because this is what id does factually...
